matplotlibcpp.h

```diff
@@ -266,7 +265,20 @@ namespace matplotlibcpp {
 		Py_DECREF(res);
 	}
 
-  
+    inline void subplot(long nrows, long ncols, long plot_number) {
```

Comment:
Nice work, thanks! But I think this function could be improved if
- there was a check that args is not nullptr after calling PyTuple_New()
- Py_DECREF(args) would be called before throwing the exception to avoid a memory leak